### PR TITLE
LF: Release LF 1.14

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN

- [LF] Add support for Exception 

CHANGELOG_END

Part of  #9914

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/spec/transaction.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Daml-LF Transaction Specification
 =================================
 
-**version 14, 04 June 2021**
+**version 14, 22 June 2021**
```
[the remark this answers is not in the document]
Comment (remyhaemmerle-da) [reply daml-lf/spec/transaction.rst:8]:
```suggestion
**version 14, 04 June 2021**
```

File: daml-lf/spec/value.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Daml-LF Value Specification
 ===========================
 
-**version 13, 06 April 2021**
+**version 14, 22 June 2021**
```
Comment (remyhaemmerle-da) [inline daml-lf/spec/value.rst:7]:
```suggestion
**version 14, 04 June 2021**
```

File: daml-lf/spec/transaction.rst [hunk 3]
```diff
@@ -183,7 +183,7 @@ later.
 +--------------------+-----------------+
 |                 13 |      2021-04-06 |
 +--------------------+-----------------+
-|       (preview) 14 |      2021-06-03 |
+|                 14 |      2021-06-03 |
```
Comment (remyhaemmerle-da) [inline daml-lf/spec/transaction.rst:186]:
```suggestion
|                 14 |      2021-06-04 |
```

File: daml-lf/spec/value.rst [hunk 4]
```diff
@@ -162,7 +162,9 @@ later.
 +--------------------+-----------------+
 |                 13 |      2021-04-06 |
 +--------------------+-----------------+
-|                dev |      2019-12-14 |
+|                 14 |      2021-22-06 |
```
Comment (remyhaemmerle-da) [inline daml-lf/spec/value.rst:165]:
```suggestion
|                 14 |      2021-04-06 |
```

File: daml-lf/spec/daml-lf-1.rst [hunk 5]
```diff
@@ -250,12 +250,12 @@ Version: 1.13
...
 
-Version: 1.14 (preview)
-.......................
+Version: 1.14
+.............
 
 * Introduction date:
 
-     2021-06-03
+     2021-06-22
```
Comment (remyhaemmerle-da) [inline daml-lf/spec/daml-lf-1.rst:258]:
```suggestion
     2021-06-04
```

File: daml-lf/spec/transaction.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (remyhaemmerle-da) [reply daml-lf/spec/transaction.rst]: Good point. I change all the date to June 4th

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel [hunk 6]
```diff
@@ -63,7 +63,8 @@ conformance_test(
     ],
     lf_versions = [
         "stable",
-        "latest",
+        # FIXME: enable the following line once canton-test-runner supports LF 1.14
+        # "latest",
```
Comment (remyhaemmerle-da) [inline ledger/ledger-api-test-tool-on-canton/BUILD.bazel:67]: cc @oliverse-da

File: daml-lf/spec/value.rst [hunk 7]
```diff
@@ -162,7 +162,9 @@ later.
 +--------------------+-----------------+
 |                 13 |      2021-04-06 |
 +--------------------+-----------------+
-|                dev |      2019-12-14 |
+|                 14 |      2021-04-06 |
```
Comment (sofiafaro-da) [inline daml-lf/spec/value.rst:165]:
```suggestion
|                 14 |      2021-06-04 |
```

**Review by sofiafaro-da — APPROVED** [review]
Looks great, thanks!

**Pull request merged**